.travis.yml

```diff
@@ -14,3 +14,11 @@ script:
     - sh build.sh
     # Invoke ci script too
     - sh contrib/ci.sh
+after_success:
+    - if [[ "$TRAVIS_TAG" != null ]]; then 
+        docker tag functions/gateway:latest-dev $DOCKER_USERNAME/gateway:latest;
```

Comment:
I think we said we wouldn't push `:latest` in the previous comments.

Reply:
We did discuss it, but wasn't sure if we came to a final resolution. I suppose, for now at least, we won't push a `:latest` tag in Travis. We may come back and add that later